src/extension/geofence/src/main/java/org/geoserver/geofence/GeofenceAccessManager.java

```diff
@@ -369,39 +404,26 @@ private AccessLimits getAccessLimits(
         return limits;
     }
 
+    private boolean allOpaque(Collection<LayerGroupContainmentCache.LayerGroupSummary> summaries) {
+        LayerGroupInfo.Mode opaque = LayerGroupInfo.Mode.OPAQUE_CONTAINER;
+        return !summaries.stream().anyMatch(gs -> !gs.getMode().equals(opaque));
```

Comment:
Why not use "allMatch" directly instead of a double negation?

Reply:
fixed